assets/components/contour/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
If this is our own helm chart, we should use v2 syntax.

Reply:
Yes this is our own helm chart.

Q: What is the motivation behind using `v2`? Is this to avoid conflict or something?

Reply:
This is to keep up to date with upstream recommendations :) if you use Helm 3.0+ CLI with `helm create`, it will generate `v2` for you.

Reply:
Oh I see. Sure making changes.

Reply:
done